pulsar-testclient/src/main/java/com/yahoo/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -62,17 +62,17 @@
         @Parameter(names = { "--conf-file" }, description = "Configuration file")
         public String confFile;
 
-        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL", required = true)
+        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL, e.g., \"ws://localhost:8080/\"", required = true)
         public String proxyURL;
 
         @Parameter(description = "/persistent/my-property/cluster1/my-ns/my-topic", required = true)
-        public String destination;
+        public List<String> destinations;
```

Comment:
If we are going to allow only one entry in the list, what is the reason to move it to a list?